docs/pt-br/actions.md

```diff
@@ -0,0 +1,185 @@
+# Ações
+
+Ações são similares às mutações, sendo que as diferenças são:
+
+- Ao invés de mudar os estados, ações cometem mutações.
+- Ações podem conter operações assíncronas arbitrárias.
+
+
+Vamos registrar uma ação simples:
+
+``` js
+const store = new Vuex.Store({
+  state: {
+    count: 0
+  },
+  mutations: {
+    increment (state) {
+      state.count++
+    }
+  },
+  actions: {
+    increment (context) {
+      context.commit('increment')
+    }
+  }
+})
+```
+
+Os handlers de ações recebem um objeto de contexto que expõe a mesma quantidade de métodos/propriedades na instância da store, portanto, você pode chamar `context.commit` para cometer uma mutação, ou acessar o estado e getters via `context.state` e `context.getters`. Vamos ver por que esse objeto de contexto não é a própria instância quando formos ver sobre [Módulos](modules.md) mais tarde.
+
+Na prática, geralmente usamos [desestruturação de argumentos](https://github.com/lukehoban/es6features#destructuring) do ES2015 para simplificar um pouco o código (especialmente quando precisamos chamar `commit` várias vezes):
+
+``` js
+actions: {
+  increment ({ commit }) {
+    commit('increment')
+  }
+}
+```
+
+### Despachando Ações
+
+Ações são disparadas com o método `store.dispatch`
+
+
+
+``` js
+store.dispatch('increment')
+```
+
+Pode parecer burrice à primeira vista: se queremos incrementar o contador, por que não simplesmente chamamos `store.commit('incrment')` diretamente? Bem, lembra que **mutações devem ser síncronas?** Ações não. Podemos realizar operações **assíncronas** dentro de uma ação.
```

Comment:
Mispealled here: `store.commit('incrment')` should be `store.commit('increment')`.